lua/lspconfig/server_configurations/julials.lua

```diff
@@ -1,5 +1,34 @@
 local util = require 'lspconfig.util'
 
+local function activate_env()
+  local bufnr = vim.api.nvim_get_current_buf()
+  local julials_client = util.get_active_client_by_name(bufnr, 'julials')
+  if not julials_client then
+    vim.notify(
+      'method julia/activateenvironment is not supported by any servers active on the current buffer',
+      vim.log.levels.WARN
```

Comment:
Probably just assert()? The command shouldn't normally be available if there are no julia clients in the buffer, right?

Reply:
`vim.notify` is a normal vim command that is available separately from the LSP client

Reply:
Yes, but when would `julials_client` ever be undefined? Commands defined in `commands` (I think?) are not defined unless their associated config is active for a buffer.

Reply:
Oh I see, sounds good!

Reply:
Revised to being an `assert`

Reply:
I am going to go ahead and revert the use of `assert` because the original comment regarding the commands only being available when the language server is active is incorrect. I can revise it back to an `assert` if this is desired.

Reply:
an assert() is sufficient.